### PR TITLE
[Bug 1223889] redirect missed product locales to English

### PR DESCRIPTION
**Pull request by MikkCZ**
This should fix in-product links for Welsh redirecting them to English. See https://groups.google.com/forum/#!topic/mozilla.dev.l10n/Mf9DTtaK7ys

CC @vesper-mozilla @rprys

### CODE REVIEW

**Comment by vesper-mozilla** [comment]
SGTM. Thank you, Michal!

Michał
vesper @ SUMO Community; IRC: vesper @ #sumo
<https://kiwiirc.com/client/irc.mozilla.org/sumo>

File: kitsune/settings.py [hunk 1]
```diff
@@ -248,6 +248,7 @@ def path(*parts):
     'bn': 'bn-BD',
     'br': 'fr',
     'csb': 'pl',
+    'cy': 'en-US',
```
Comment (glogiotatidis) [inline kitsune/settings.py]: Please set this to `None` to get the default language, which is `en-US`

**Comment by glogiotatidis** [comment]
Merged here https://github.com/mozilla/kitsune/commit/02810ee8964b1dc120c2142379ee70889bd64ad2

**Comment by glogiotatidis** [comment]
Thanks! 👍

**Pull request closed without merging**